src/test/java/jeryl/fyp/logic/commands/DeleteCommandTest.java

```diff
@@ -17,19 +18,24 @@
 import jeryl.fyp.model.ModelManager;
 import jeryl.fyp.model.UserPrefs;
 import jeryl.fyp.model.student.Student;
+import jeryl.fyp.model.student.StudentId;
 
 /**
  * Contains integration tests (interaction with the Model) and unit tests for
  * {@code DeleteCommand}.
  */
 public class DeleteCommandTest {
 
+    final StudentId STUDENT_ID = new StudentId("A0123456G");
```

Comment:
This should be put in `CommandTestUtil` and used in a similar way `VALID_*_AMY` variables are used